ReadMe.md

```diff
@@ -2,14 +2,14 @@
 
 [![Discord](https://img.shields.io/discord/740930220399525928.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](http://flipperzero.one/discord)
 
-<img src="https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png" />
+![](https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png)
 
 Welcome to [Flipper Zero](https://flipperzero.one/)'s Firmware repo!
 Our goal is to create nice and clean code with good documentation, to make it a pleasure for everyone to work with.
 
 # Update firmware
 
-<a href="https://update.flipperzero.one/">Get Latest Firmware from Update Server</a>
+[Get Latest Firmware from Update Server](Get Latest Firmware from Update Server)
```

Comment:
Hi.

Not an image.

Reply:
I made a mistake and copied text instead of link